_posts/2022-05-27-jay-lofstead.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "SC22 Mini-series: Inclusivity Committee"
+author: "@mmundt"
+rse: "Jay Lofstead"
+excerpt: "Jay Lofstead of Sandia National Laboratories has some big plans for the Inclusivity Committee at Supercomputing 2022."
+date: "2022-05-27 8:30:00"
+external_media: 
+length: 16624071
+duration: "00:18:47"
+explicit: "no"
+resources:
+  - name: Supercomputing 2022
+    url: https://sc22.supercomputing.org/
+  - name: SC22 Inclusivity Committee
+    url: https://sc22.supercomputing.org/planning-committee/#Inclusivity
+  - name: SC22 Inclusivity Contact
+    url: sc-inclusivity@info.supercomputing.org
```

Comment:
We probably want this to be a mailto: link. You can also just add the email to the episode notes if you want to show (but not link) it.